django_docker_engine/container_managers/docker_engine.py

```diff
@@ -30,6 +31,10 @@ class PossiblyOutOfDiskSpace(DockerEngineManagerError):
     pass
 
 
+class GetContainerTimeout(DockerEngineManagerError):
```

Comment:
Fine... though it sounds like you're naming it for a method name, but it's not the method it's thrown in?